ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/insert-values.json

```diff
@@ -15,7 +15,7 @@
         {"topic": "test_topic", "timestamp": 1234, "key": "key", "value": {"ID": 10}}
       ],
       "responses": [
-        {"admin": {"@type": "currentStatus", "statementText": "{STATEMENT}"}}
+        {"admin": {"@type": "currentStatus"}}
```

Comment:
We shouldn't really be removing this.. it's testing the API is returning the correct statement text with the response.  

Tricky...